sr_robot_launch/launch/srhand.launch

```diff
@@ -77,7 +79,6 @@
   
   <!-- Trajectory controller -->
   <group if="$(arg hand_ctrl)">  
-    <node name="$(arg hand_id)_trajectory_controller" pkg="sr_utilities" type="trajectory_controller_spawner.py"
-          args="$(arg hand_trajectory)" />
+    <node name="$(arg hand_id)_trajectory_controller" pkg="sr_utilities" type="trajectory_controller_spawner.py"/>
```

Comment:
the hand_traj parameter should be private (see comment on https://github.com/shadow-robot/sr_core/pull/16)